rosetta/Gruntfile.js

```diff
@@ -107,13 +94,12 @@ module.exports = function(grunt) {
   grunt.loadNpmTasks('grunt-contrib-copy');
   grunt.loadNpmTasks('grunt-contrib-watch');
   grunt.loadNpmTasks('grunt-concurrent');
-  grunt.loadNpmTasks('grunt-bowercopy');
   grunt.loadNpmTasks('grunt-json-server');
   grunt.loadNpmTasks('grunt-browserify');
   grunt.loadNpmTasks('grunt-mocha-phantom-istanbul');
 
   grunt.registerTask('lint', ['eslint']);
-  grunt.registerTask('build', ['copy:app', 'bowercopy']);
+  grunt.registerTask('build', ['copy:app']);
   grunt.registerTask('test', ['browserify:coverage', 'mocha']);
```

Comment:
The reason for the test to fail is that they refer to non-existing files in bower_components. We need to modify file rosetta/test/load.js